functions/src/express.ts

```diff
@@ -7,13 +7,22 @@ import * as functions from "firebase-functions";
 import express from "express";
 import jwt from "express-jwt";
 import jwksRsa from "jwks-rsa";
+import * as Sentry from "@sentry/node";
 import cors from "cors";
 
 const app = express();
 
+//setup sentry
+if (functions.config().sentry && functions.config().sentry.dns) Sentry.init({ dsn: functions.config().sentry.dns });
```

Comment:
I think typescript's optional chaining could help here!
```suggestion
if (functions.config()?.sentry?.dns) Sentry.init({ dsn: functions.config().sentry.dns });
```

Reply:
Yessirrrrr Ima add that as well as the default Sentry.catchException thing we talked about in the meeting